HHS/OPRE-OPS
# feat: ✨ adds filter CANList Pills (Tags)

## What changed

- adds pills for CANsList filters

## Issue

#347 #2909

## How to test

1. goto CANs list
2. apply filters
3. see pill per filter
4. remove pill
5. filters keep in sync
6. table is updated

## Screenshots

https://github.com/user-attachments/assets/75bbc3ba-b47b-480c-8830-a78c11e3c9b4

## Definition of Done Checklist
